docs/src/views/Colors.tsx

```diff
@@ -0,0 +1,352 @@
+            specific to the colors.
+          </p>
+          <p>
+            There are two things important in order for colors to work transparently when the theme
+            switching is in play:
+          </p>
+          <ul>
+            <li>
+              <Text weight="bold" color="primary">
+                {link('Color palette', '#color-palette')}
+              </Text>{' '}
+              - central place for all colors available in the application,
+            </li>
+            <li>
+              <Text weight="bold" color="primary">
+                {link('Color scheme', '#color-scheme')}
+              </Text>{' '}
+              - design tokens for all colors used in the application that should be appropriately
+              mapped in all themes.
+            </li>
+          </ul>
+          <p>
+            Be aware that everything that follows is our recommendation, not the requirement
+            (everything will work even if you'll decide to structure the palette and schemas
+            differently in your theme).
+          </p>
+          <Header as="h2" content="Color palette" />
+          <p>
+            Colors in {link('Teams color palette', '/color-palette')} have the following
+            categorization.
+          </p>
+
+          <Header as="h3">Primitive colors</Header>
+          <p>
+            This part of the palette contains colors that, semantically, cannot have any tints. This
+            group is represented by two colors, {code('black')} and {code('white')} - as there is
+            nothing blacker than black and nothing whiter than white.
+          </p>
+
+          <Grid columns={2}>
+            {_.map(['black', 'white'], color => (
+              <div key={color}>
+                <ColorBox
+                  name={color}
+                  rounded
+                  size="big"
+                  value={colors[color]}
+                  copyToClipboardIcon={false}
+                />
+              </div>
+            ))}
+          </Grid>
+
+          <Header as="h3">Natural colors</Header>
+          <p>
+            This part of palette includes colors from those that are the most commonly used among
+            popular frameworks ({code('blue')}, {code('green')}, {code('grey')},{code('orange')},{' '}
+            {code('pink')}, {code('purple')}, {code('teal')}, {code('red')}, {code('yellow')}). Each
+            color includes at least ten gradients, this allows us to satisfy most common needs.
+          </p>
+          <p>
+            This decision is experienced from Material UI and allows to define more variants than by
+            using semantical naming ({code('lightest')}, {code('lighter')}, etc.). However, there is
+            no requirement for client to define all the gradient values for each color - it is just
+            enough to define those that are actually used in the app.
```

Comment:
```diff
-  - it is just enough to define those that are actually used in the app
```

Do we really need this phrase?

Reply:
I wanted to emphasize on the fact that in the color palette you should have only values that are used, and not add values just so that you will have 10 values for each color.